.nix/config.nix

```diff
@@ -10,8 +10,6 @@
       hierarchy-builder.override.version = "coq-elpi-2";
       hierarchy-builder-shim.job = false;
 
-      coq-elpi.override.version = "master";
```

Comment:
@gares this was asking the action to test the master branch explicitly.

Reply:
Thanks